meta-lmp-base/conf/distro/lmp-wayland.conf

```diff
@@ -0,0 +1,9 @@
+require conf/distro/lmp.conf
+
+DISTRO = "lmp-wayland"
+DISTROOVERRIDES = "lmp:lmp-wayland"
+DISTRO_NAME = "Linux-microPlatform Wayland"
+
+DISTRO_FEATURES_append = " wayland opengl"
+
+IMAGE_LINGUAS ?= "en-us"
```

Comment:
This is already set in lmp.config and with the same value, probably not needed.

Reply:
Good catch!  Fixing.